modules/nf-core/vuegen/environment.yml

```
@@ -0,0 +1,7 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+channels:
+  - bioconda
+  - conda-forge
+dependencies:
```

```suggestion
dependencies:
 - python>=3.11
```
would that not work?

---

No, I tried that and got this error on the pre-commit; apparently, conda just allows to define a specific version.

```
Schema validation errors were encountered.
  modules/nf-core/vuegen/environment.yml::$.dependencies[1]: 'python>=3.11' is not valid under any of the given schemas
  Underlying errors caused this.

  Best Match:
    $.dependencies[1]: 'python>=3.11' is not of type 'object'
  Best Deep Match:
    $.dependencies[1]: 'python>=3.11' does not match '^.*[^><]=[^><].*$'
```

---

I would then pin to the minimal viable version.